app/services/twilio_api_service.rb

```diff
@@ -1,8 +1,5 @@
-class NotificationService
```

Comment:
I refactored this class a while ago to be more object oriented and to avoid having to rescue twilio errors outside of this class. Looks like I'm coming full circle. The reason I'm choosing to allow the errors to bubble up again is to better match the pattern @rsanheim recommended for handling Imo invitations.